app/helpers/orchid/application_helper.rb

```diff
@@ -87,6 +87,59 @@ def clear_search_text
     return new_params
   end
 
+  # Creates classes to populate the <html> element.
+  #   Use @page_classes to add custom classes separated by space
+  #
+  # By default outputs:
+  #   [custom classes]        string passed in by opt. @page_classes variable
+  #   section_[@section]      if @section
+  #   controller_[controller] if params[:controller]
+  #   action_[action]         if params[:action]
+  #   page_[page]             orchid page if applicable (browse, item, etc)
+  def html_classes
+    classes = []
+
+    if @page_classes.present?
+      classes << @page_classes
+    end
+
+    if @section.present?
+      classes << "section_#{@section}"
+      # add additional section for home page backwards compatibility
+      classes << "section_home" if current_page? home_path
```

Comment:
Do we have some stuff baked in pretty widely that this one needs to remain for a while?

Reply:
This might be a question for @karindalziel , but I erred on the side of caution because I know that quite often she designs home pages much differently than the other pages

Reply:
@karindalziel thought that she probably has used this previously, but since it would only be on one or two apps at this point, I'll see if it's used in any of the orchid styles and if not, just kill it off.  If a home page of an app needs something more specific than the controller / action information, it can pass a class in.